activestorage/lib/active_storage/errors.rb

```diff
@@ -26,4 +26,7 @@ class FileNotFoundError < Error; end
 
   # Raised when a Previewer is unable to generate a preview image.
   class PreviewError < Error; end
+
+  # Raised when a checksum is not supported by a service
+  class UnsupportedChecksumError < Error; end
```

Comment:
Conventionally it should be named `ChecksumUnsupportedError` to follow general convesion "what? kind?" like "RecordNotFound", "InputInvalid" etc.

Reply:
Removing as this is likely not needed until the direct upload implementation.